acceptance/tests/facts/windows.rb

```diff
@@ -22,7 +22,10 @@
     os_version  = '2012 R2'
     kernel_version = '6.3'
   elsif agent['platform'] =~ /-10/
-    os_version  = /^10\./
+    os_version  = '10'
```

Comment:
Actually I just found it [here](https://github.com/puppetlabs/facter/blob/2.x/lib/facter/operatingsystem/windows.rb). If I'm reading this right, it looks like we never updated it for Windows Server 2016, same as Facter 3. So, given the output you pasted above, @MosesMendoza , does Windows 10 Enterprise not count as a consumer release? And if not, what do we want to return for it?